test/Engine/FFITarget/TestDefaultArgumentAttributes.cs

```diff
@@ -16,5 +18,10 @@ public static double ComputeCircleConflict([DefaultArgument("Point.ByCoordinates
         {
             return radius * radius * Math.PI;
         }
+
+        public static IEnumerable<Point> ComputeFlattenedPoints([DefaultArgument("FFITarget.DesignScript.Point.ByCoordinates((1..10)<1>, (2..20)<2>, 0)")]IEnumerable<IEnumerable<Point>> points)
```

Comment:
is this from this PR?

Reply:
This isn't related to the PR. I just thought it would be useful to add a test with replication guides in default argument attributes. Will submit this together with the test case in a separate PR.